onnxruntime/test/onnx/TestCaseResult.h

```diff
@@ -25,30 +27,32 @@ enum class EXECUTE_RESULT {
 
 class TestCaseResult {
  public:
-  TestCaseResult(size_t count, EXECUTE_RESULT result, const std::string& node_name1) : node_name(node_name1), excution_result_(count, result) {
-    ::onnxruntime::SetTimeSpecToZero(&spent_time_);
+  TestCaseResult(size_t count, EXECUTE_RESULT result, const std::string& node_name1)
+      : node_name(node_name1), execution_result_(count, result) {
+    onnxruntime::SetTimeSpecToZero(&spent_time_);
   }
 
   void SetResult(size_t task_id, EXECUTE_RESULT r);
 
   const std::vector<EXECUTE_RESULT>& GetExcutionResult() const {
-    return excution_result_;
+    return execution_result_;
   }
 
   //Time spent in Session::Run. It only make sense when SeqTestRunner was used
-  ::onnxruntime::TIME_SPEC GetSpentTime() const {
+  onnxruntime::TIME_SPEC GetSpentTime() const {
     return spent_time_;
   }
 
   //Time spent in Session::Run. It only make sense when SeqTestRunner was used
-  void SetSpentTime(const ::onnxruntime::TIME_SPEC& input) const {
+  void SetSpentTime(const onnxruntime::TIME_SPEC& input) const {
     memcpy((void*)&spent_time_, &input, sizeof(input));
   }
 
   //only valid for single node tests;
   const std::string node_name;
 
  private:
-  ::onnxruntime::TIME_SPEC spent_time_;
-  std::vector<EXECUTE_RESULT> excution_result_;
+  onnxruntime::TIME_SPEC spent_time_;
+  std::vector<EXECUTE_RESULT> execution_result_;
+  std::mutex result_mutex_;
```

Comment:
std::mutex might be just fine, but i noticed that OrtMutex is used elsewhere. is one of them preferred?

Reply:
I think we should be consistent and use OrtMutex everywhere. there are quite a few places where an update is needed (not just the test projects) so I'll create a separate PR for that cleanup.